src/siddhant/Shop.java

```diff
@@ -0,0 +1,67 @@
+package siddhant;
+
+public class Shop{
+	int maggie;
+	int dosa;
+	int masala;
+	int panipuri;
+	int pouches;
+	void setinitial(int maggie1, int dosa1, int masala1, int panipuri1, int pouches1){
+		maggie = maggie1;
+		dosa = dosa1;
+		masala = maggie1;
+		panipuri = panipuri1;
+		pouches = pouches1;
+	}
+	void Requestedquantity(int reqmaggie1,int reqdosa1, int reqmasala1, int reqpanipuri1, int reqpouches1){
+		if(reqdosa1<=dosa){
+			dosa=dosa - reqdosa1;
+		}else{
+			System.out.println("Sorry not sufficient quantity we can only give you : " +dosa);
+			System.out.println("Product " +dosa+ " running out of stock");
+			dosa =0;
```

Comment:
If request stock is not available mean we are setting direct it zero, 
what if second time your calling same method and let say you request 20 packs, then this looks not logically correct